contrib/src/main/java/com/datatorrent/contrib/hbase/AbstractHBaseAppendOutputOperator.java

```diff
@@ -20,15 +20,17 @@
 
 import java.io.IOException;
 
-import org.apache.hadoop.hbase.client.Append;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import org.apache.hadoop.hbase.client.Append;
```

Comment:
Why did the import order change? Is the new order correct as per the import-order guideline or was the old one better?